olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -507,7 +507,7 @@ spec:
                 serviceAccountName: knative-operator
                 initContainers:
                   - name: cli-artifacts
-                    image: registry.ci.openshift.org/openshift/knative-v1.0.0:kn-cli-artifacts
+                    image: registry.ci.openshift.org/openshift/knative-v1.0.1:kn-cli-artifacts
```

Comment:
I can't pull this image as of now. So it was probably not built by ci-operator? Perhaps the code changes were published under `registry.ci.openshift.org/openshift/knative-v1.0.0:kn-cli-artifacts` because the kn client doesn't have a specific branch for 1.0.1 ?

Reply:
I have no idea. Reverting to the original image.